kbfsmd/root_metadata_v2.go

```diff
@@ -94,13 +93,13 @@ type WriterMetadataExtraV2 struct {
 	codec.UnknownFieldSetHandler
 }
 
-// BareRootMetadataV2 is the MD that is signed by the reader or
+// RootMetadataV2 is the MD that is signed by the reader or
 // writer. Unlike RootMetadata, it contains exactly the serializable
 // metadata.
-type BareRootMetadataV2 struct {
+type RootMetadataV2 struct {
```

Comment:
Why this name change?

Reply:
The 'Bare' prefix in libkbfs was intended to distinguish between just the serializable type BareRMD and the annotated type RMD with ExtraMetadata, PrivateMetadata, and associated TlfHandle that's used by most of libkbfs. Since only the Bare version is used in kbfsmd (and on the server side), there's no need for the prefix when it's moved there.

There's some chance of confusion between `kbfsmd.RootMetadata` and `libkbfs.RootMetadata`, but I don't think it's a big deal.